docs/feature-gates.md

```diff
@@ -341,3 +342,15 @@ More documentation will be coming in the future.
 
 User should provide a file server with this feature, and store its authentication credential in a Secret. Antrea
 Controller is required to be configured with the permission to read the Secret.
+
+### Multicluster
+
+`Multicluster` feature on agent side enables Antrea Multi-cluster Gateways which route Multi-cluster Service
```

Comment:
Maybe we can add a link `[Antrea Multi-cluster Gateways](multicluster/user-guide.md#multi-cluster-gateway-configuration)`

Reply:
Done